client/src/backoffice/InboxListPage.tsx

```diff
@@ -123,13 +127,15 @@ const InboxListPage: FC = () => {
   const handleNotificationTypeSelect = (e: React.ChangeEvent<HTMLSelectElement>) => {
     const { value } = e.target;
 
+    if (value === '') handleRadio('notificationType', value);
     if (value === 'inbox') handleRadio('notificationType', value);
     if (value === 'push') handleRadio('notificationType', value);
   };
 
   const handleRecipientSelect = (e: React.ChangeEvent<HTMLSelectElement>) => {
     const { value } = e.target;
 
+    if (value === '') handleRadio('recipientFilter', value);
     if (value === 'everyone') handleRadio('recipientFilter', value);
     if (value === 'event') handleRadio('recipientFilter', value);
   };
```

Comment:
If the 3 `if` are calling the same function `handleRadio('recipientFilter', value);` why are we asking for the value?
Maybe I'm missing something but anyways we can improve this a little bit:

```typescript
const handleRecipientSelect = (e: React.ChangeEvent<HTMLSelectElement>) => {
  const { value } = e.target;
  const valuesToHandle = ['', 'everyone', 'event'];

  if (valuesToHandle.includes(value)) handleRadio('recipientFilter', value);
};
```